safe-global/web-core
# fix: check if all tokens are hidden on dashboard

## What it solves

Resolves #1607

## How this PR fixes it

The hiding of native tokens is no longer possible.

## How to test it

1. Open a Safe with multiple tokens and observe that the "Tokens" amount on the dashboard switches from the loading skeleton to the number of tokens owned.
2. Navigate to the assets page and hide all tokens then back to the dashboard and refresh. Observe that the "Tokens" amount changes from the loading skeleton to 0.
3. Switch to a Safe that has fewer tokens (hidden token amount is higher). Observe that the the loading skeleton switches from loading skeleton to 0.

Varing amounts of hidden tokens should no make the loading skeleton change to 0 to n. It should always change from the skeleton to the amount of tokens not hidden.

## Screenshots

![hidden-tokens](https://user-images.githubusercontent.com/20442784/215434980-d43a90bd-fde6-4218-a90b-bcbe8b062fe5.gif)
